release/test_releases_json.sh

```diff
@@ -187,4 +196,20 @@ function _test_releases_json_get_liferay_upgrade_folder_version {
 		"${2}"
 }
 
-main "${@}"
\ No newline at end of file
+function _test_releases_json_tag_jakarta_product_versions {
+	local product_group_version="${1}"
+
+	local product_group_version_json="${product_group_version//./-}.json"
```

Comment:
@GabrielFernandoS, can you please replace it with a more legible code as the following?
```
local product_group_version_json="$(echo "${product_group_version}" | tr '.' '-').json"
```